packages/office-ui-fabric-react/src/components/Tooltip/examples/Tooltip.NoScroll.Example.tsx

```diff
@@ -9,7 +9,7 @@ export class TooltipNoScrollExample extends BaseComponent<{}> {
   public render(): JSX.Element {
     return (
       <TooltipHost content="This is the tooltip" id={this.tooltipId} tooltipProps={{ style: { overflowY: 'auto' } }}>
-        <DefaultButton>Tooltip without scroll</DefaultButton>
+        <DefaultButton aria-describedby={this.tooltipId}>Tooltip without scroll</DefaultButton>
```

Comment:
Why is this one `aria-describedby` instead of `aria-labelledby`?

Reply:
ooops, should have been labelledby!